src/app/workflow/workflow.component.html

```diff
@@ -136,15 +139,23 @@ <h3>
   <div class="side-block col-md-2 col-lg-3 ng-scope" *ngIf="isWorkflowPublic">
     <div class="panel panel-default" *ngIf="workflow">
       <div class="panel-heading">
-        <h3>Versions</h3>
+        <h3>Recent Versions</h3>
       </div>
       <div class="panel-body">
-        <div class="input-group col-md-12">
-          <div class="form-group">
-            <select name="workflowVersions" class="form-control select-box" [(ngModel)]="selectedVersion">
-              <option *ngFor="let workflowVersion of workflow.workflowVersions" [ngValue]="workflowVersion">{{workflowVersion.name}}</option>
-            </select>
-          </div>
+        <div>
+          <span *ngFor="let sortedVersion of getSortedVersions(workflow.workflowVersions, defaultVersion);">
```

Comment:
Avoid using a function in places like this.

Reply:
actual question, is this a style recommendation, a performance thing?

Reply:
The function gets ran every time the view gets checked/updated.  On my computer, it gets ran about 40+ times.

Reply:
https://stackoverflow.com/questions/42066490/call-a-component-method-from-html-in-angular2